docs/issues.md

```diff
@@ -1,3 +1,15 @@
 # Known issues
 
-?> *TODO* Placeholder for known issues
+The following open issues might affect your modules.
+## Autoscaling issue with IBM Cloud Databases (ICD) and v1.50 of the IBM Cloud Provider
+
+You receive a failure when you try to deploy any IBM Cloud Databases (ICD) module that includes the autoscaling block and uses IBM Cloud Provider version 1.50 or later.
+
+### Why it's happening
+
+An issue exists with the IBM Cloud Provider version 1.50. For more information, see [issue 4339](https://github.com/IBM-Cloud/terraform-provider-ibm/issues/4339) in the Terraform Provider GitHub repo.
+
+### How do I solve this problem?
+
+- Set the `IBM-Cloud/ibm"` provider to use only [version 1.49](https://github.com/IBM-Cloud/terraform-provider-ibm/releases/tag/v1.49.0).
+- Make sure that properties that need to be set and that are passed from your deployment to the ICD module are not set to null or empty values.
```

Comment:
Hmm, I think we have a workaround for this now. But I think it still has to be rolled out to our ICD modules. Maybe we go with this for now, but update (or remove) when we roll the workaround out.
@shemau anything else to add here?

Reply:
@ocofaigh - workaround documented here is for consumer wanting to use auto-scaling capability in the module.
The workaround we discussed last week with @shemau is to prevent the issues from happening to consumer for the module who do not use auto-scaling capabilities.

Reply:
Thanks @shemau. My intent is to reword this to indicate that if you see the error, make sure that you're on the latest release of the module first. If you still have the problem, then use the workarounds about the provider version and properties.